invenio_communities/assets/semantic-ui/js/invenio_communities/requests/requests_items/ComputerTabletRequestsItems.js

```diff
@@ -0,0 +1,75 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import _truncate from "lodash/truncate";
+import React from "react";
+import RequestTypeLabel from "@js/invenio_requests/request/RequestTypeLabel";
+import RequestStatusLabel from "@js/invenio_requests/request/RequestStatusLabel";
+import { RequestActionController } from "@js/invenio_requests/request/actions/RequestActionController";
+import { Icon, Item } from "semantic-ui-react";
+import { RightBottomLabel } from "./RightBottomLabel";
+
+
+export const ComputerTabletRequestsItems = ({
+  result,
+  community,
+  differenceInDays,
+  isCreatorCommunity,
+  creatorName,
+  updateQueryState,
+  currentQueryState,
+}) => {
+  const refresh = () => {
+    updateQueryState(currentQueryState);
+  };
+  return (
+    <Item key={community.id} className="community-item computer tablet only flex">
```

Comment:
lack of division tablet/desktop to separate components, like you have in other PRs